# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cool-retro-term/cwctype.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- cool-retro-term-1.1.1/qmltermwidget/lib/TerminalCharacterDecoder.cpp.vanilla	2020-07-23 17:54:12.508700905 +0200
+++ cool-retro-term-1.1.1/qmltermwidget/lib/TerminalCharacterDecoder.cpp	2020-07-23 17:55:09.703699118 +0200
@@ -31,6 +31,8 @@
 // Konsole
 #include "konsole_wcwidth.h"
 
+#include <cwctype>
+
 using namespace Konsole;
 PlainTextDecoder::PlainTextDecoder()
  : _output(0)
